adversarial_workflow/cli.py

```
@@ -3255,15 +3265,45 @@ def main():
+            if not local_evaluators:
+                print(f"{RED}Error: No evaluators installed.{RESET}")
+                print("Install evaluators with: adversarial library install <name>")
+                return 1
+
+            if evaluator_override not in local_evaluators:
+                print(f"{RED}Error: Evaluator '{evaluator_override}' not found.{RESET}")
+                print()
+                print("Available evaluators:")
+                # Show unique evaluators (avoid duplicates from aliases)
+                seen = set()
+                for name, cfg in sorted(local_evaluators.items()):
```

Ruff (0.14.14)

[warning] Loop control variable `name` not used within loop body

(B007)